src/iris/ui/templates/application.html

```diff
@@ -121,6 +121,24 @@ <h3>
     {{/if}}
     <br>
 
+    {{#if @root.isInViewMode}}
+      <span class="light">Custom email "from" address: </span>
+      {{#if custom_sender_addresses.email}} 
+        <span class="label label-info">{{custom_sender_addresses.email}} <i class="glyphicon" data-variable="{{custom_sender_addresses.email}}"></i></span> 
+      {{else}}
+        <span class="light">None (using default sender address)</span>
+      {{/if}}
+    {{else}}
+      <span class="light">custom email "from" address:</span>
+      {{#if custom_sender_addresses.email}}
+      <span class="label label-info">{{custom_sender_addresses.email}} <i class="glyphicon glyphicon-remove remove-address" data-variable="{{custom_sender_addresses.email}}"></i></span> 
```

Comment:
This label/badge flow is confusing for an element that only takes a single value. We should switch to a text field more like the templates and less like the context variables. The top bar is also getting crowded, so we may want to move this and "supported modes" into a "Notification settings" subheader.

Reply:
Changed UI to separate it out into its own div.